docker-compose.yml

```diff
@@ -4,21 +4,23 @@
   todoapp-net:
 services:
   pythonapp:
-    image: todoapp:2.0.0
+    image: niklafinskiy/todoapp:2.1.0
     build:
       context: .
       dockerfile: Dockerfile
     ports:
       - "8080:8080"
+    env_file:
+      - .env.todoapp
     environment:
-      - PYTHONUNBUFFERED=1
+      - PYTHONBUFFERED=1
     networks:
       - todoapp-net
     depends_on:
       - mysql
     restart: unless-stopped
   mysql:
-    image: mysql-local:1.0.0
+    image: niklafinskiy/mysql-local:1.0.0
     build:
```

Comment:
Similar to the `pythonapp` service, the `mysql` service also has both `build` and `image` specified. Consider removing the `image` line if you intend to build the image locally.